extensions/amp-analytics/analytics-vars.md

```diff
@@ -361,3 +371,75 @@ Provides the number of seconds that have elapsed since 1970. (Epoch time)
 
 Example value: `1452710304312`
 
+## Visibility Variables
+
+### backgrounded
+
+A binary variable with possible values of 1 and 0 to indicate that the page/tab was sent to background at any point before the hit was sent. 1 indicates that the page was backgrounded while 0 indicates that the page has always been in the forground.
+
+### backgroundedAtStart
+
+A binary variable with possible values of 1 and 0 to indicate that the page/tab backgrounded at the time when the page was loaded. 1 indicates that the page was loaded in the background while 0 indicates otherwise.
+
+### continuousVisibleTime
```

Comment:
Wondering if it makes sense to call this maximumContinuousVisibleTime because it's the maximum. Theoretically you can also provide the minimum (non-zero) value, too.